bigswitch-patch-image.sh

```diff
@@ -2,6 +2,8 @@
 
 set -euxo pipefail
 
+IMAGE_PATH=${1:-"/home/stack/overcloud-full.qcow2"}
```

Comment:
According to https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux_OpenStack_Platform/7/html/Director_Installation_and_Usage/sect-Creating_Directories_for_Templates_and_Images.html, the images are at /home/stack/images, Let's follow the same pattern.